lib/domains/character/types.tsx

```diff
@@ -182,14 +184,37 @@ export type IImageBlock = {
   value: string;
 };
 
+export type ILinkBlock = {
+  type: BlockType.Link;
+  meta: {
+    helperText?: string;
+    displayName?: string;
+    editName?: boolean;
+  };
+  value: {
+    link: string;
+    isValid: boolean;
+  };
+};
+
+export type ISeparatorBlock = {
+  type: BlockType.Separator;
+  meta: {
+    helperText?: string;
+    hasLabel?: boolean;
```

Comment:
**question** Usually, I use the function `previewContentEditable` and if the result of that function is empty, I hide the label when the advanced mode is toggled off. Do you think we could do the same thing here or were there any particular reasons you think necessitated the use of a particular meta property here ?

Reply:
Nope. I think that will work 👍

Reply:
So testing out the code change now, it looks like the only minor benefit we get from having something stored like this, is that a in advanced mode a user can 'toggle' the visibility of the label input, so that it can be hidden in and out of advanced mode. 

That isn't the current convention right now for other blocks, so I'm happy to make this line up a bit more with that : )